jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/RetainableByteBuffer.java

```diff
@@ -2101,6 +2101,10 @@ private boolean shouldAggregate(RetainableByteBuffer buffer, long size)
 
             if (_minRetainSize == -1)
             {
+                // If it is a chunk, then retain
+                if (buffer instanceof Content.Chunk)
```

Comment:
I suspect this isn't always the best thing to do, but it's going to be hard to figure out when it is and isn't.